requirements-base.txt

```diff
@@ -17,6 +17,12 @@ enum34>=1.1.6,<1.2.0
 exam>=0.5.1
 functools32>=3.2.3,<3.3
 futures>=3.2.0,<4.0.0
+google-cloud-bigquery>=1.3.0,<1.4.0
```

Comment:
We don't need bigquery here at all.

Can we just move all of these Google libs into optional?